rinja_derive/Cargo.toml

```diff
@@ -14,7 +14,9 @@ rust-version = "1.65"
 proc-macro = true
 
 [features]
-config = ["serde", "basic-toml"]
+config = ["serde", "toml"]
```

Comment:
I think the use of `serde` and `basic-toml` are implementation details, so the line should read `config = ["dep:serde", "dep:basic-toml"]`, without exposing them as features.

Reply:
It's what I did originally but `cargo` was unhappy with `feature = "basic-toml"`. Hence why I went full in.

Reply:
Oh, yeah. I guess *all* usages of `#[cfg(feature = "toml")]`, `#[cfg(feature = "basic-toml")]`, and `#[cfg(feature = "serde")]` are wrong in `rinja_derive`, and should be `#[cfg(feature = "config")]` instead.

Reply:
Now that makes more sense. Changing to that then!